src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/disabled_form_alert.html.twig

```diff
@@ -25,6 +25,9 @@
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
 {% block content %}
+  <div class="header-toolbar d-print-none">
+    {{ multistore_controller.header(false).getContent()|raw }}
```

Comment:
I'm afraid it's not the right way to do it.
Could you please create a Twig function to do it, it will be easier to use, easier to maintain.

Reply:
I'm not sure I understand what you want to centralize, you mean a function for when we want to call a method from a controller, that would take a controller as a parameter ?